lib/chef/knife/config_list.rb

```diff
@@ -128,8 +127,7 @@ def render_table(profiles, padding: 1)
         else
           table.render do |renderer|
             renderer.border do
-              mid   "-"
-              style :green
```

Comment:
Somehow `TTY::Table` unable to add border-color for some OS. Keeping plain border here.